crates/aggchain-proof-contracts/src/config.rs

```diff
@@ -0,0 +1,46 @@
+use alloy::primitives::{address, Address};
+use prover_utils::from_env_or_default;
+use serde::{Deserialize, Serialize};
+
+/// Address of the `GlobalExitRootManagerL2SovereignChain.sol` contract
+/// on the L2 chain is always fixed.
+const GLOBAL_EXIT_ROOT_MANAGER_L2_SOVEREIGN_CHAIN_ADDRESS: Address =
+    address!("0xa40D5f56745a118D0906a34E69aeC8C0Db1cB8fA");
+
+/// Address of the `PolygonRollupManager.sol` contract
+/// on the L1 chain.
+const POLYGON_ROLLUP_MANAGER: Address = address!("0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e");
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+#[serde(rename_all = "kebab-case")]
+pub struct AggchainProofContractsConfig {
+    #[serde(alias = "PolygonRollupManagerContract")]
+    pub polygon_rollup_manager_contract: Address,
+    #[serde(alias = "GlobalExitRootManagerL2SovereignChainContract")]
```

Comment:
Why is there an `alias` ?
Should we define a `#[serde(default = "default_global_exit_root_manager_v2_sovereign_chain")]` ?

Reply:
So I am not sure about that, maybe contracts should be referred with their original name to prevent confusion?

Reply:
The agglayer-node approach is outdated and we had this alias for backward compatibility with the go-version.

Reply:
Ok, lets switch then to `kebab-case`.